pub.sh

```
@@ -1,7 +1,9 @@
-set -ex
+set -x
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)